pkg/query/flamegraph_table.go

```diff
@@ -543,15 +543,14 @@ func trimFlamegraphNodes(ctx context.Context, tracer trace.Tracer, nodes []*quer
 		if node.Cumulative < threshold {
 			continue
 		}
+		var oldChildren FlamegraphChildren = node.Children
```

Comment:
```suggestion
		oldChildren := node.Children
```

Reply:
I was trying to do this but was unable to as `node.Children` ie `[]*querypb.FlamegraphNode` is not an interface. 

So, I created an alias type `FlamegraphChildren` and added some utility functions like below:

```
type FlamegraphChildren []*querypb.FlamegraphNode

func (n FlamegraphChildren) Cumulative() int64 {
	cumulative := int64(0)
	for _, child := range n {
		cumulative += child.Cumulative
	}
	return cumulative
}
```

I was unable to typecast and access these utility functions, so defined a variable with this type and used those for accessing the functions. 

There could a different way for this problem, please let me know.

Reply:
ah, then what you have is fine, another option is:
```
oldChildren := FlamegraphChildren(node.Children)
```

Reply:
Cool, thanks for the tip!